pkgs/by-name/li/libcdio/package.nix

```diff
@@ -17,10 +17,11 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "libcdio";
   version = "2.2.0";
 
-  src = fetchFromGitHub {
-    owner = "libcdio";
-    repo = "libcdio";
-    tag = finalAttrs.version;
+  src = fetchzip {
+    urls = [
+      "https://github.com/libcdio/libcdio/archive/refs/tags/${finalAttrs.version}.tar.gz"
+      "https://git.savannah.gnu.org/cgit/libcdio.git/snapshot/libcdio-${finalAttrs.version}.tar.gz"
```

Comment:
We cannot do that, as the auto generated tar ball is not guaranteed to be stable. You need to use an artifact.

Reply:
The hashes won't change, so @toastal is right here, but see https://github.com/NixOS/nixpkgs/pull/396510#pullrequestreview-2776895001 for why this is still a bad idea.

Reply:
Oh derp. The hash of the archive is not stable but since we unpack it, it is not a problem since the content is.